tests/integration/kubernetes/confidential_common.sh

```diff
@@ -52,6 +52,19 @@ function check_hypervisor_for_confidential_tests() {
     fi
 }
 
+# This function verifies whether the input hypervisor supports confidential tests and 
+# relies on `KATA_HYPERVISOR` being an environment variable
+function check_hypervisor_for_confidential_tests_tee_only() {
```

Comment:
Optional suggestion to refactor `check_hypervisor_for_confidential_tests` to call this method for the tee stuff.

Reply:
Done.